Composer/packages/client/src/components/ManageService/ManageService.tsx

```diff
@@ -213,33 +153,7 @@ const ManageService: React.FC<ManageServiceProps> = (props: ManageServiceProps)
           setSubscriptionsErrorMessage(err.message);
         });
     }
-  }, [token]);
-
-  const hasAuth = async () => {
-    let newtoken = '';
-    if (userShouldProvideTokens()) {
-      if (isShowAuthDialog(false)) {
-        setShowAuthDialog(true);
-      }
-      newtoken = getTokenFromCache('accessToken');
-      if (newtoken) {
-        const decoded = decodeToken(newtoken);
-        if (decoded) {
-          setToken(newtoken);
-          setUserProvidedTokens(true);
-        } else {
-          setTenantsErrorMessage(
-            formatMessage(
-              'There was a problem with the authentication access token. Close this dialog and try again. To be prompted to provide the access token again, clear it from application local storage.'
-            )
-          );
-        }
-      }
-    } else {
-      setUserProvidedTokens(false);
-    }
-    setCurrentStep('subscription');
-  };
+  }, [isAuthenticated]);
```

Comment:
You reference currentUser, does that need to be in the dependencies?

Reply:
isAuthenticated and currentUser are set in tandem.  if isAuthenticated is true, currentUser is set. So I thought it made more sense semantically to have it fire when the authentication status changed rather than changes to the user model...